include/ice-engine/threadpool.h

```diff
@@ -0,0 +1,49 @@
+#ifndef THREADPOOL_H
+#define THREADPOOL_H
+//#include "common.h"
```

Comment:
is there a reason this is commented out? if it was causing build issues that might be a problem we need to look at.